activejob/lib/active_job/queue_adapters/sidekiq_adapter.rb

```diff
@@ -37,7 +37,7 @@ class JobWrapper #:nodoc:
         include Sidekiq::Worker
 
         def perform(job_data)
-          Base.execute job_data
+          Base.execute job_data.merge('provider_job_id' => jid)
```

Comment:
If this is being done here is it still needed to set the `provider_job_id` in the `enqueue` method?